Formula/json-fortran.rb

```diff
@@ -1,8 +1,8 @@
 class JsonFortran < Formula
   desc "Fortran 2008 JSON API"
   homepage "https://github.com/jacobwilliams/json-fortran"
-  url "https://github.com/jacobwilliams/json-fortran/archive/4.3.0.tar.gz"
-  sha256 "c97f8de53e2ca9ee91fb3148bfa971b00eaea6e757b5e3d67cc4b4ff197e392e"
+  url "https://github.com/jacobwilliams/json-fortran/archive/5.0.0.tar.gz"
+  sha256 "0a1bf8788bdf8bdc6af009ae078ed374b2c95e0cfc3f30354b5dc5d3d35e5e66"
```

Comment:
I may split this out  into a separate PR, in hopes that  it can get merged before this PR